scripts/check.sh

```diff
@@ -3,27 +3,27 @@ set -e
 
 case $1 in
     "docs")
-        cargo run --bin kagami -- docs | diff - docs/source/references/config.md || {
+        cargo run --release --bin kagami -- docs | diff - docs/source/references/config.md || {
             echo 'Please re-generate docs using `cargo run --bin kagami -- docs > docs/source/references/config.md`'
```

Comment:
then I guess you should update the `echo` message?

Reply:
Or better yet, autogenerate it based on the command. 

In `bash` you can evaluate a string by enclosing it in `$()`.

Reply:
thank you for catching.